ssds/__init__.py

```diff
@@ -118,10 +118,8 @@ def _check_name_exists(self, submission_id: str, name: Optional[str]) -> str:
         return name
 
     def _compose_ssds_key(self, submission_id: str, submission_name: str, path: str) -> str:
-        path = path.strip("/")
-        dst_prefix = f"{submission_id}{self._name_delimeter}{submission_name}"
-        ssds_key = f"{dst_prefix}/{path}"
-        blobstore_key = f"{self.prefix}{ssds_key}"
+        ssds_key = f"{submission_id}{self._name_delimeter}{submission_name}/{path.strip('/')}"
```

Comment:
Might be good to have a docstring here about the key naming conventions used here, or if this information is elsewhere, a link to the documentation.

Reply:
Also, `delimeter` -> `delimiter`

Reply:
Already documented in readme.

spelling fixed.

Reply:
Actually, I'll need to fix the spelling error in a future PR